pkg/kubelet/sysctl/whitelist.go

```diff
@@ -48,7 +48,7 @@ func NewWhitelist(patterns []string) (*patternWhitelist, error) {
 	}
 
 	for _, s := range patterns {
-		if !policyvalidation.IsValidSysctlPattern(s) {
+		if !policyvalidation.IsValidSysctlPattern(s, true) {
 			return nil, fmt.Errorf("sysctl %q must have at most %d characters and match regex %s",
 				s,
 				validation.SysctlMaxLength,
```

Comment:
must match `policyvalidation.SysctlContainSlashPatternFmt`

Reply:
I mean the error message this prints needs to change to reference SysctlContainSlashPatternFmt

Reply:
ok, got it.